EpikCord/client/websocket_client.py

```diff
@@ -265,7 +308,111 @@ def stop_loop_on_completion(f: asyncio.Future):
             pass
         finally:
             future.remove_done_callback(stop_loop_on_completion)
-            self.utils.cleanup_loop(loop)
+            Utils.cleanup_loop(loop)
+
+
+
+    async def _voice_server_update(self, data: discord_typings.VoiceServerUpdateEvent):
+        voice_data: discord_typings.VoiceServerUpdateEvent = data["d"]
+        payload = {
+            "token": voice_data["token"],
+            "endpoint": voice_data["endpoint"],
+            "guild_id": voice_data["guild_id"],
+        }
+
+        if voice_data["endpoint"]:
+            payload["endpoint"] = voice_data["endpoint"]
+
+        return await self.dispatch("voice_server_update", payload)
```

Comment:
`return` is useless in this context (end of function) and it may break the typing as python typing will interpret the function return to not be set or not be the expected type.

Reply:
Agreed